### ISSUE TITLE
Setup.hs does not compile with Cabal > 2.0.1.1

### ISSUE THREAD

**Issue by eexexiexit** [johannesgerer/blazeT#2]
Problem: 

> /home/user/Packages/Haskell/Own/teilfeil-0.1.0.2/dist-newstyle/tmp/src-16404/blazeT-0.0.5/dist/setup/setup.hs:11:9:                           
> error:                                                                                                                                        
> Not in scope: ‘haddockHscolour’                                                                                                               
> Perhaps you meant ‘haddockHscolourCss’ (imported from                                                                                         
> Distribution.Simple.Setup)                                                                                                                    
> )                      
> 

Reason: package Cabal:
Distribution.Simple.Setup.HaddockFlags
does not anymore have the field:
haddockHscolour.

Solution:

Remove corresponding line in Setup.hs

**Comment by habibalamin** [comment]
@eexexiexit see discussion on #1.

TL;DR I've forked the package, and the maintainer has given me access to upload new versions to Hackage, and permission to point all links to my fork, which you can find on my GitHub profile. I've uploaded 0.0.6 to incorporate the patches necessary to get this working on more recent versions of GHC/`base` and Cabal.
